### DISCUSSION TITLE
求个gkd保活模块

### DISCUSSION THREAD

**Discussion by BaiLai7513** [gkd-kit/gkd#741] [Announcements]
### 新功能描述

求个gkd保活模块适配magisk kernelsu apatch，还有种方案适配dhizuku

**Comment by longzho** [comment]
![Screenshot_2024-11-18-22-31-10-39_befd3b55c4270226022f86e9f9cafb33.jpg](https://github.com/user-attachments/assets/41be99d8-224c-416f-b415-1602740006db)

![Screenshot_2024-11-18-22-31-32-88_befd3b55c4270226022f86e9f9cafb33.jpg](https://github.com/user-attachments/assets/f36f6ee5-7395-43ed-bda6-84b9038bdc89)

![Screenshot_2024-11-18-22-31-49-38_baaa51257c82a8b3f281ce09e6f2493f.jpg](https://github.com/user-attachments/assets/e36b28ba-2f7c-46ab-a1cc-469d08035e98)

后台耗电的权限也全部给够的话，保活不用愁的吧？(记得锁定)

**Comment by longzho** [comment]
![Screenshot_2024-11-19-13-28-50-48_9d26c6446fd7bb8e41d99b6262b17def.jpg](https://github.com/user-attachments/assets/04dd1368-0a22-49f7-ab25-f1b1a91e4299)

爱玩机好像有？看到有人装了
但鄙人没试过
你可以试试

**Reply by China54188** [reply]
shizuku应该就能保活啦，有root就刷模块就成

**Comment by China54188** [comment]
用高级授权 不要普通授权普通授权它会掉
